generate.go

```diff
@@ -1,7 +0,0 @@
-// +build generate
-
-package generate
-
-//go:generate go run github.com/operator-framework/operator-sdk/cmd/operator-sdk generate k8s
-//go:generate go run github.com/operator-framework/operator-sdk/cmd/operator-sdk generate crds
-//go:generate ./create-api-docs.sh
```

Comment:
You probably didn't intend to remove the API documentation generation for Antora, as it's unrelated to the Operator SDK.
However, that API doc generator is buggy. K8up has a different tool:
* https://github.com/vshn/k8up/blob/master/generate.go#L13
* https://github.com/vshn/k8up/tree/master/docs

I propose to re-add resp. replace the API doc-gen in a separate PR.